victor_hardware_interface/SunriseWorkspace/UofMR1_2017_LeftBottom/src/armlab/lcm/robotInterface/LCMRobotInterface.java

```diff
@@ -259,11 +256,11 @@ private SmartServo createSmartServoMotion(final joint_path_execution_parameters
     {
         SmartServo motion = new SmartServo(iiwa7_arm_.getCurrentJointPosition());
         motion.setMinimumTrajectoryExecutionTime(MINIMUM_TRAJECTORY_EXECUTION_TIME);
-        motion.setJointVelocityRel(params.joint_relative_velocity);                    
+        motion.setJointVelocityRel(params.joint_relative_velocity);
         motion.setJointAccelerationRel(params.joint_relative_acceleration);
         motion.overrideJointAcceleration(params.override_joint_acceleration);
         motion.setTimeoutAfterGoalReach(TIMEOUT_AFTER_GOAL_REACH);
-        
+        motion.setSpeedTimeoutAfterGoalReach(0.1);

```

Comment:
What does this do? My guess would be if after 0.1 seconds we send another target without specifying velocity, it will default to zero?